src/data/faq_de.json

```
@@ -136,10 +136,10 @@
                         "textblock": [
-                            "Es gibt derzeit (Stand: 9 Juni 2021) noch keine Impfzertifikate mit QR-Codes, die mit der App eingescannt werden können.",
-                            "Ein in der Vergangenheit und momentan ausgestelltes Dokument ist trotz des QR-Codes kein gültiges digitales Impfzertifikat, sondern lediglich eine Impfbescheinigung, die dem deutschen Infektionsschutzgesetz folgt, nicht aber den aktuellen neuen abgestimmten europäischen Richtlinien für ein digitales Impfzertifikat.",
-                            "Diese neuen digitalen Impfzertifikate werden demnächst ausgestellt. Solche digitalen Zertifikate können nur von autorisierten Stellen über die zentrale Schnittstelle bereitgestellt werden, die im Auftrag der Bunderegierung von anderen Partnern entwickelt wurde. Der Feldtest dafür wurde inzwischen beendet, und die Funktion wird nun flächendeckend ausgerollt.",
-                            "Es ist noch nicht abschließend geklärt, wer genau neben den Impfzentren auch in der Lage sein wird, ein Impfzertifikat auf Basis eines Impfnachweises, d.h. dem gelben Impfausweis bzw. der oben genannten Impfbescheinigung, auszustellen. Das Netz wird stufenweise weiter aufgebaut."
+                            "Es gibt derzeit (Stand: 10 Juni 2021) nur einige wenige Stellen, von denen Bescheinigungen mit QR-Codes ausgegeben werden, die mit der App eingescannt werden können.",
+                            "Ein in der Vergangenheit und aktuell ausgestelltes Dokument ist in der Regel noch kein gültiges digitales Impfzertifikat, sondern lediglich eine Impfbescheinigung nach §22 des deutschen Infektionsschutzgesetzes, die noch nicht den aktuellen neuen abgestimmten europäischen Richtlinien für ein digitales Impfzertifikat folgt. Daher kann man dieses Dokument, wie auch die QR-Codes und Barcodes im gelben Impfausweis mit der Corona-Warn-App <b>nicht</b> einscannen.",
+                            "Diese neuen digitalen Zertifikatsbescheinigungen (die gescannt werden können) werden ab 14. Juni von autorisierten Stellen über einen zentralen Dienst bereitgestellt, der im Auftrag der Bunderegierung entwickelt wurde. Der Feldtest dafür wurde inzwischen beendet, und die Funktion wird nun flächendeckend ausgerollt. Weitere Informationen zu möglichen lokalen Standorten finden Sie hier:  <a href='https://www.aponet.de/artikel/corona-digitaler-impfnachweis-ab-14-juni-in-apotheken-24262' target='_blank' rel='noopener noreferrer'>Corona: Digitaler Impfnachweis ab 14. Juni in Apotheken</a>.",
+                            "Zum Prozess, wie ein Impfzertifikat auf Basis eines Impfnachweises (gelber Impfausweis bzw. der Impfbescheinigung) ausgestellt wird, gibt es  <a href='https://digitaler-impfnachweis-app.apps.public.bfarm.de/impfzertifikat-ausstellen/' target='_blank' rel='noopener noreferrer'>hier</a> weitere Informationen. Das Netz lokaler Aussteller wird stufenweise weiter aufgebaut."

```

Remove one space to much

```suggestion
                            "Zum Prozess, wie ein Impfzertifikat auf Basis eines Impfnachweises (gelber Impfausweis bzw. der Impfbescheinigung) ausgestellt wird, gibt es <a href='https://digitaler-impfnachweis-app.apps.public.bfarm.de/impfzertifikat-ausstellen/' target='_blank' rel='noopener noreferrer'>hier</a> weitere Informationen. Das Netz lokaler Aussteller wird stufenweise weiter aufgebaut."
```

---

thanks will correct later.

---

sorry

---

Will open PR in a few minutes.